component-samples/rv/src/main/java/org/fido/iot/sample/RvAppSettings.java

```diff
@@ -24,4 +24,10 @@
   // EPID URL
   public static final String EPID_URL = "epid_online_url";
   public static final String EPID_TEST_MODE = "epid_test_mode";
+
+  // ondie settings
+  public static final String ONDIE_CACHEDIR = "ondie_cache";
```

Comment:
Originally in the JSON based implementation, the OnDie cache was consumed by Manufacturer and Owner only, mainly for checking revocation.
1) Is the FDO OnDie implementation not the same as the SDO OnDie implementation?
2) If it is the same then these properties are not required for RV server

Reply:
The RV uses the cache indirectly to determine if a cert chain is OnDie or not by comparing the Root CA against the known root CAs in OnDie. There are obtained from the cache but we could specify the bytes directly. I'll do some more investigation .

Reply:
Reworked the dependencies so RV (and TO0_TO1) components do not require any OnDieCache or configuration.